<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userReportview</class>
 <widget class="QWidget" name="userReportview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>病人评价</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btSave">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QComboBox" name="resultBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>好</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>差</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="dnameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>诊断时间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="commentBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>好</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>差</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>治疗效果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>医生名</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="conditionEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>201</width>
     <height>66</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>就诊情况</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancle">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
